spec/util/app-service-mock.js

```diff
@@ -85,6 +85,9 @@ MockAppService.prototype.listen = function(port) {
 };
 
 MockAppService.prototype._trigger = function(eventType, content) {
+    if (content.user_id) {
```

Comment:
Matrix used to use `user_id` instead of sender in the good ol days. Bridge transactions currently come down in both formats from the homeserver, so while `user_id` works, we should use `sender`. For the sake of not having to change all the tests and code at the same time, we've bodged in this change to ensure both keys exist in tests.